src/checker.rs

```diff
@@ -139,6 +144,17 @@ mod test {
         assert!(check(&rules, &String::from("yes!")));
     }
 
+    #[test]
+    fn test_max_characters() {
+        let rules : Rules = Rules {
+            max_characters: 25,
+            ..Default::default()
+        };
+
+        assert!(!check(&rules, &String::from("not less than or equal to 25")));
+        assert!(check(&rules, &String::from("less than or equal to 25")));
```

Comment:
I'm not seeing anything obviously wrong here, can you have a deeper look into why this test case fails? Might also be some default rule?

Reply:
I'm not sure either. I was playing with MAX and 999 but could not pass the tests yet (my first lines of code in Rust).

Reply:
Let's see if it is fixed here also.